wordpressorg.test/provision/wp-config.php

```diff
@@ -55,6 +55,11 @@
 define( 'WPORG_THEME_DIRECTORY_BLOGID',  35 );
 define( 'WPORG_PLUGIN_DIRECTORY_BLOGID', 367 );
 define( 'WPORG_TRANSLATE_BLOGID',        351 );
+define( 'WPORG_SUPPORT_FORUMS_BLOGID',    368 );
+
+define( 'WP_CORE_LATEST_RELEASE',        "5.4" );
+define( 'RECOMMENDED_PHP',         		 7.1 );
+
```

Comment:
I think PHPCS would tweak the spacing/tabs a bit, but definitely not a blocker.

Reply:
I'll clean this up in a PR.